server/constants/consts.ts

```diff
@@ -0,0 +1,5 @@
+export const config = {
+    DATABASE_LOCAL: 'mongodb://127.0.0.1:27017/scafeinder',
+    DATABASE_CLOUD: '',
+    PORT: ''
+};
```

Comment:
Please add empty line in the end of file.

Reply:
done